karma.conf.js

```diff
@@ -0,0 +1,46 @@
+require('./esbuild.js');
+const path = require('path');
+
+const globalAny = global;
+globalAny.global = globalAny;
+
+module.exports = function (config) {
+    config.set({
+        browsers: ['ChromeHeadless'],
+        frameworks: ['mocha', 'chai', 'snapshot', 'mocha-snapshot'],
+        reporters: ['spec'],
+        files: [
+            'packages/**/*.test.ts',
```

Comment:
Files should have .spec.tsx extension as we decided in the doc. Why did you choose .test.?

Also, there are extensions that are not needed. 

BTW, why do we need to add:
 'packages/**/*/lib/index.js',
            './__snapshots__/**/*.md',

Reply:
I used .test. because I was foreseeing issues differentiating integration and UT tests, both for people looking at the files as for having to set ignores in configs. I just find it clearer imo, but we can debate it.

1) Stupid Karma needed to know where that file was or it would break
2) Karma needs to know which kind of files to include for snapshot testing

Reply:
About .test., spec was choose for consistency, so tests are spec files, if we have spec and test... how do I know which one is integration or which one is unit test?, both indicates me that they are test.

the way to differentiate them is with the .integration. which is included in the integration test files

Regarding Karma config, all good